amundsen_application/api/metadata/v0.py

```diff
@@ -553,3 +553,34 @@ def _log_update_table_tags(*, table_key: str, method: str, tag: str) -> None:
         logging.exception(message)
         payload = jsonify({'msg': message})
         return make_response(payload, HTTPStatus.INTERNAL_SERVER_ERROR)
+
+
+# TODO: Implement real support
+@metadata_blueprint.route('/user', methods=['GET'])
+def get_user() -> Response:
+    try:
+        user_id = get_query_param(request.args, 'user_id')
```

Comment:
what is user id? and what will the profile page url look like?

cc @ttannis @danwom

Reply:
Currently all the user node is indexed with email address, it would be good if the FE send the email(convert the user id into email) before sending the request to metadata.

Reply:
This is whatever value we will use to identify the user. It can be anything, even based on some existing parameter like email. The url will look like `amundsen.lyft.net/user/:user_id`

Reply:
cool 👍

Reply:
My hope was that we would set an explicit id parameter on the User node even if internally we choose to set it to the email.